### DISCUSSION TITLE
Check contribution from `ZZTo2L2Nu, WWTo1L1nu2Q, WZTo1l1nu2Q and tZq production` and summarize there contribution.

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#25] [Comments_HIG_23_001] [answered]
Tab 8: It looks to me like some final states are missing: You would presumably also have a background from ZZTo2L2Nu, WWTo1L1nu2Q, WZTo1l1nu2Q. Did you check?

Tab 8: what about e.g. tZq production? Do you consider this too?

**Comment by jialin-guo1** [comment]
I am going to submit crab jobs to produce these samples
```shell
/ZZTo2L2Nu_TuneCP5_13TeV_powheg_pythia8/RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v1/MINIAODSIM
/WWTo1L1Nu2Q_4f_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v2/MINIAODSIM
/tZq_ll_4f_ckm_NLO_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL18MiniAODv2-106X_upgrade2018_realistic_v16_L1v1-v2/MINIAODSIM
```

**Comment by jialin-guo1** [comment]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/509b9559-e4e3-4acc-9546-45fa9eab45db)
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/e7647add-d2b9-46fe-9a35-eedb36f327ac)


**with these two plots, I think we can answer why we don't include those three process**
**they are massZZ distribution in signal region in resolved and merged case separately**

**Reply by ram1123** [reply]
Can you add all samples them separately. For now you merged some of them.

**Reply by jialin-guo1** [reply]
no I did not merged them it just hard to see 

<img width="797" alt="image" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/76bfa564-f1ca-4333-872c-18aecf18f249">

**Comment by ram1123** [comment]
In the legend why you write (TT AND WW)?

**Reply by jialin-guo1** [reply]
TT and WW are what we always used samples. "ZZTo2L2Nu,tZq,WWTo1L1Nu2Q" are new added sample according to conveners' comments

**Reply by jialin-guo1** [reply]
anyway, I can separate them

![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/1e7220ba-009d-4c99-8495-b9f417692d57)
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/62e1b11f-e1f6-4769-a0c3-29e5663e1a47)


N(WWTo2L2Nu) = 45.6 VS N(WWTo2L1Nu2Q)=0.95

**Reply by jialin-guo1** [reply]
for physic point of view. WWTo1LNu2Q process is hard to pass our Z boson cut. Because one Nu will not identify to a lepton. I think that is also the reason that I don't use it even in 2016public

**Reply by ram1123** [reply]
What about the `WZTo1l1nu2Q` sample? Did you also add this?

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/d09b746d-d321-4b9d-add9-98acde7fc6f3)

now I added WZTo1l1nu2Q

**Reply by ram1123** [reply]
@jialin-guo1 These numbers are for inclusive ggH + VBF?

**Reply by jialin-guo1** [reply]
these numbers are not apply any VBF of b-tag cut. yes inclusive one

**Reply by ram1123** [reply]
after PN cut?

**Reply by jialin-guo1** [reply]
no this is resolved case plot. I made a wrong tittle somehow

**Reply by ram1123** [reply]
You computed this only for the resolved category?

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/e7647add-d2b9-46fe-9a35-eedb36f327ac)

this is merged

**Reply by ram1123** [reply]
For the merged case I don't see number for

1. WW -> 2l2nu
2. WZ -> lnuqq

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/5cc0e3be-c1f8-465c-976e-8afeb36d80cb)

![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/1f778ab9-7e30-4525-be45-b7a36bf4b838)

these resolved and merged case plot will be more clear

**Reply by ram1123** [reply]
For WZ, in the merged case, no event passed. Or it is zero because of rounding off to 1 decimal place?

**Reply by jialin-guo1** [reply]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/e03e7d09-f25e-4814-94ae-db0194951e4b)

in think because of no event passed. since the number still zero even I round of to .4f

**Comment by ram1123** [comment, marked as answer]
|Merged case | Resolved case |
|--- |--- |
|<img width="360" alt="image" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/5220316/7f233f06-1b2f-4af2-931e-4d8976c2c559"> |<img width="360" alt="image" src="https://github.com/ram1123/2l2Q_limitSettingTool/assets/5220316/255ee279-7108-4db8-a680-7b8e44596935">|


---


Sample  | Yield - Resolved category | Fraction of yield w.r.t. total  - Resolved category | Yield - Merged category | Fraction of yield w.r.t. total - Merged category
-- | -- | -- |-- | -- | 
Z+jets | 65273.97 | 91.71% | 1565.27 | 71.92%
ZV | 2932.38 | 4.12% | 530.92 | 24.40%
tbar{t} | 2855.46 | 4.01% | 68.20 | 3.13%
tZq | 50.91 | 0.07% | 8.88 | 0.41%
WW → l+ ν l- ν | 45.60 | 0.06% | 2.25 | 0.10%
ZZ → l± l∓ ν ν | 12.93 | 0.02% | 0.48 | 0.02%
WW → l± ν qq | 0.95 | 0.00% | 0.26 | 0.01%
WZ → l ν qq | 0.19 | 0.00% | 0.00 | 0.00%

**Reply by ram1123** [reply]
# Added to the appendix of AN

![स्क्रीनशॉट 2023-08-17, अ 6 43 16 पर](https://github.com/ram1123/2l2Q_limitSettingTool/assets/5220316/1bd29ffe-a6b3-4e02-82f1-b6ea38a37aab)

**Comment by jialin-guo1** [comment]
with veto additional loose lepton

resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/eba21f01-6bc4-4b61-bb92-563310b7f96e)

merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/bff45f44-9cda-49c1-8345-75b976323c0f)

**Reply by ram1123** [reply]
Did you also compute the significance value?

**Comment by jialin-guo1** [comment]
WWTo1L1Nu2Q:   WWTo1L1Nu2Q_4f_TuneCP5_13TeV-amcatnloFXFX-pythia8  51.65 
tZq:    tZq_ll_4f_ckm_NLO_TuneCP5_13TeV-amcatnlo-pythia8   0.07561 
ZZTo2L2Nu:    ZZTo2L2Nu_TuneCP5_13TeV_powheg_pythia8  0.6008 
WZTo1L1Nu2Q:    WZTo1L1Nu2Q_4f_TuneCP5_13TeV-amcatnloFXFX-pythia8  9.119

**Comment by jialin-guo1** [comment]
I update WZTo3LNu_TuneCP5_13TeV-amcatnloFXFX-pythia8 number here:

**without loose lepton veto**
resolved: 4.311
merged:  0.140

**with loose lepton veto**
resolved: 2.958
merged: 0.079

**Reply by ram1123** [reply]
We don't have to apply the lepton veto.
